karma.conf.js

```diff
@@ -7,12 +7,16 @@ module.exports = function(config) {
             'karma-browserify',
             'karma-mocha',
             'karma-mocha-reporter',
+            'karma-junit-reporter',
             'karma-firefox-launcher'
         ],
         preprocessors: {
             'test/*.js': ['browserify']
         },
-        reporters: ['mocha'],
+        reporters: ['mocha', 'junit'],
+        junitReporter: {
```

Comment:
Should the junit reporter only be used during CI? Mainly asking since some people may not be OK with this file getting created when these tests are run locally. Alternatively, add `*test-results.xml` to `.gitignore`.